README.md

```diff
@@ -1,83 +1,139 @@
-# libp2p Docs
+<p align="center">
+<img align="center" src="static/images/libp2p-docs-logo.png" width="615">
+</p>
+
+<div align="center">
+<h2> The networking layer of the web3 and the distributed web</h2>
```

Comment:
Can we remove this? libp2p is used much broader than just in web3.

Reply:
addressed